.editorconfig

```diff
@@ -0,0 +1,14 @@
+# EditorConfig is awesome: http://EditorConfig.org
```

Comment:
Makes Idea and other editors recognise the style used in the project. Very useful.